chat-plugins/tcg.js

```diff
@@ -20,7 +20,13 @@ var tcgsearch = function (target, room, user, cmd, self) {
 			response += data;
 		});
 		a.on('end', function () {
-			var result = JSON.parse(response);
+			var result;
+
+			try {
+				result = JSON.parse(response);
+			} catch (e) {
+				return self.sendReply("ERROR: Could not parse query:" + e);
```

Comment:
I'm not sure if it's a good idea to send the error details as a reply to the user.
Maybe log to the console?